CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+## [1.2.1] - 2017-07-19
```

Comment:
You are certainly not the first nor the last to make that mistake, you get one free pass on it and then I ask you to fix it up next time. At some point I am gonna work on improving the PR templates to have better instructions to help people with this.